src/Data/Nat/Base.agda

```diff
@@ -130,6 +132,47 @@ instance
 >-nonZero⁻¹ : ∀ n → .{{NonZero n}} → n > 0
 >-nonZero⁻¹ (suc n) = z<s
 
+-- The property of being a non-zero, non-unit
+
+pattern 2+ n = suc (suc n)
+
+trivial : ℕ → Bool
+trivial 0      = true
+trivial 1      = true
+trivial (2+ _) = false
+
+NonTrivial : Pred ℕ 0ℓ
+NonTrivial = T ∘ not ∘ trivial
+
+instance
+  nonTrivial : ∀ {n} → NonTrivial (2+ n)
+  nonTrivial = _
+
+pattern 1<2+n {n} = s<s (z<s {n})
```

Comment:
Again should go up with the definition of `<` I think?

Reply:
One question which has been in my mind: should its argument be explicit or implicit, and hence, what even should this be called? As it stands, or else
```agda
pattern 1<2+ n = s<s (z<s {n})
```

Reply:
To be consistent I'd advocate for `z<ss` and leave it implicit?